pkg/security/scc/byrestrictions.go

```diff
@@ -35,9 +36,19 @@ const (
 	runAsRangePoints   points = 20000
 	runAsUserPoints    points = 10000
 
+	runCapsDefaultPoints  points = 5000
+	runCapsAddOnePoints   points = 300
```

Comment:
I don't want to create an impression that it's just the number of caps that matters, that they are all equal. Adding a capability makes the SCC allow things that are not expected by default -- that makes it mentally much more permissive and that should be easily "cleared" by dropping one different capability.

Of course, we can have discussions whether the proposed ration 300 / 50 is the right one.